user-tools/README-CVImage.md

```diff
@@ -1,10 +1,39 @@
+## Bot Segment Type
 
-# How to Create a CV Image Bot Segment
+###  CVImage
 
-1. Use a screenshot tool to capture the target area of your game and save this as a JPG file.
- - For this example we saved the file as `change_profile_button.jpg`
- 
-![Change Profile Button](./sample_images/change_profile_button.jpg)
+Disclaimer: The `CVImage` type is still in an experimental phase and may provide inconsistent or unexpected results in many situations.  We are continuing to evaluate and tune this type.  See the [Limitations and Notes](#limitations-and-notes) section below for more information.
+  - The `CVImage` type can be used in Bot Segments for both `endCriteria` and/or `botAction`.  This type looks for the presence of the supplied image in the current frame of the game.  This type communicates with our remote AIService instance to perform the evaluation.  When a result is found, the AIService provides the bounding rect information to our SDK to confirm the existence and location of the supplied image in the current frame.
```

Comment:
`in the current frame` -> if this retries until it detects the image, we should make that explicit

Reply:
updated for image and text